project/frontend/src/components/SGonksLists/LongSGonksList/LongSGonksList.js

```diff
@@ -3,18 +3,22 @@ import classes from "./LongSGonksList.module.css";
 import LongSGonkRow from "./LongSGonkRow/LongSGonkRow";
 
 const LongSGonksList = (props) => {
-  const investmentRows = props.investments.map((investment, i) => {
-    return (
-      <LongSGonkRow
-        //TODO replace key with investment id once it's ready
-        key={i}
-        searchTerm={investment.searchItem}
-        buyInDate={investment.dateInvestedMilliSeconds}
-        amountInvested={investment.amtInvested}
-        currentValue={investment.currentValue}
-      ></LongSGonkRow>
-    );
-  });
+  const investmentRows = !props.investments ? (
```

Comment:
(I know this file is in flux, feel free to disregard this comment)

I think again ternaries aren't helping readability here - would be clearer to use
```js
let investmentRows = <div>undefined</div>
if (props.investments !== undefined) {
  investmentRows = props.investments.map((investment, i) => {
    // ...
}
```

Reply:
I will add a TODO comment in there to fix in the PR which deals with displaying partial data!

Reply:
Done.